tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -492,7 +492,7 @@ qos/test_qos_sai.py::TestQosSai:
   skip:
     reason: "Unsupported testbed type."
     conditions:
-      - "topo_name not in ['ptf32', 'ptf64', 't0', 't0-64', 't0-116', 't0-35', 'dualtor-56', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-backend', 't2', 't2_2lc_36p-masic', 't2_2lc_min_ports-masic'] or asic_type in ['mellanox']"
```

Comment:
![image](https://user-images.githubusercontent.com/46945843/189756289-dd2a2b9f-f694-470c-b489-b11f8c1e0716.png)

Unable to run qos test case unless i change to above.

Reply:
It should work. In your docker under tests/.pytest_cache/v/BASIC_FACTS can you please check the values of topo_name and topo_type for your testbed.  For one of our testbeds we see:

```
  "topo_name": "t2_ixre_chassis8", 
  "topo_type": "t2"
```

Reply:
![image](https://user-images.githubusercontent.com/46945843/190277215-0b278e5e-4a1d-41c7-80b7-b59c7f47ef25.png)

Reply:
![image](https://user-images.githubusercontent.com/46945843/190278863-e29eccb7-a98d-4678-8979-28680d08244c.png)

Reply:
This is working case - it is showing error but still going thru 
![image](https://user-images.githubusercontent.com/46945843/190278978-336b617d-d4ea-496f-b606-ebf610d4e1c7.png)

Reply:
It seems to be skipping test with your changes. Please check.

Reply:
@vmittal-msft  - Could you please try the following changes to tests_mark_conditions.yaml for TestQosSai

```
qos/test_qos_sai.py::TestQosSai:
  skip:
    reason: "Unsupported testbed type."
    conditions:
      - "topo_name not in ['ptf32', 'ptf64', 't0', 't0-64', 't0-116', 't0-35', 'dualtor-56', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-backend'] or asic_type in ['mellanox']"
      - "topo_type not in ['t2']"
```

Reply:
Reverted our changes to use topo_type and use topo_name only